### ISSUE TITLE
top level assertions don't include a stack trace

### ISSUE THREAD

**Issue by mvdan** [frankban/quicktest#79]
https://play.golang.org/p/T7QxpYdwfhg

```
=== RUN   TestStandalone
    quicktest.go:348: 
        error:
          values are not equal
        got:
          "a"
        want:
          "b"
        stack:
```

Note how the stack seems to be missing. This does not happen if I use `qt.New` instead.

**Comment by frankban** [comment]
Thanks for reporting!

**Comment by frankban** [comment]
This is now released as `v1.11.1`.
